container/worker/run_openqa_worker.sh

```diff
@@ -9,8 +9,15 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+  groupmod -g "$group" --non-unique kvm
+  [[ $(getent group kvm | cut -d ":" -f3) -ne "$group" ]] && { echo "Unable to ensure the ID of the kvm matches the group ID of /dev/kvm. KVM will not work."; exit 1; } 
   usermod -a -G kvm _openqa-worker
+  qemu-kvm -S &
+  qemu_pid=$!
+  sleep 1
+  (pgrep qemu-system-x86 >/dev/null) || (echo "Unnable to start qemu-kvm"; exit 1)
```

Comment:
I'd just get completely rid of this. If the line above fails these lines won't be executed anyways due to `set -e`. So just `kill $!` should be sufficient.